test/IDE/local_types.swift

```diff
@@ -121,18 +121,18 @@ public var singlePattern: Int {
 
 // Cannot be public, because inlinable default arguments cannot contain local types
 func singleDefaultArgument(i: Int = {
-  //CHECK-DAG: 10LocalTypes21singleDefaultArgument1iySi_tFfA_SiycfU_06SingledE6StructL_V
+  //CHECK-DAG: 10LocalTypes21singleDefaultArgument1itSi_tFfA_SiycfU_06SingledE6StructL_V
```

Comment:
These are USRs, `iy` is correct here (and in the next two)

Reply:
Oh those were not intentional by me, sorry for missing those... thanks!